infra/app/env-template/main.tf

```
@@ -54,6 +54,12 @@ module "database" {
   admin_password = module.database_password.random_password
 }
 
+module "doc_upload" {
+  source           = "../../modules/doc-upload-s3"
+  environment_name = var.environment_name
+  service_name     = local.analytics_service_name
```

Let's add a new local for this. Something like: `doc_upload_s3_name = "${local.project_name}-doc-upload-${var.environment_name}"` and add it around line 25 and then use it here:

```suggestion
  service_name     = local.doc_upload_s3_name
```